tests/Infrastructure/Behat/ApiFeatureContext.php

```diff
@@ -20,4 +31,17 @@ public function cleanEnvironment()
         $this->connections->clear();
         $this->connections->truncate();
     }
+
+
+    /** @AfterStep */
+    public function publishEvents()
```

Comment:
Here we can see the clients of the `SymfonySyncDomainEventPublisher` which I was commenting before. In this client we can see the implications of not composing the `SymfonySyncDomainEventPublisher` with the actual `SymfonySyncEventBus`.

I mean, we're lying with the semantics as soon as the `SymfonySyncDomainEventPublisher#publishRecorded` method doesn't publish anything. I'm for avoiding this kind of confusion just composing the `SymfonySyncDomainEventPublisher` with the `SymfonySyncEventBus` making them interact while calling to the `SymfonySyncDomainEventPublisher#publishRecorded` method avoiding possible confusions and having to implement the very same lines we're seeing right below this comment on every single `SymfonySyncDomainEventPublisher` class.

What do you think? Am I missing something?

Thanks!!